php/composer.json

```diff
@@ -9,7 +9,7 @@
     "php": ">=7.0.0"
   },
   "require-dev": {
-    "phpunit/phpunit": ">=5.0.0"
+    "phpunit/phpunit": ">=5.0.0 <8.5.27"
```

Comment:
`>=` [is not a version constraint](https://thephp.cc/articles/the-death-star-version-constraint).